### ISSUE TITLE
Add a LICENSE

### ISSUE THREAD

**Issue by ericdallo** [gufoe/text-translator#45]
Hi, Can you please add a LICENSE to the project?
Thanks!

**Comment by jtojnar** [comment]
It will also need to be confirmed by all the past contributors whose code is still in the tree:

https://github.com/gufoe/text-translator/graphs/contributors

**Comment by gufoe** [comment]
@jtojnar how do I ask all of them? Should I email everybody?

**Comment by jtojnar** [comment]
I would ping them here, asking them to comment agreeing to the license similarly as done here: https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/merge_requests/397#note_438286

When they do not reply within a reasonable period of time, I would try contacting them via e-mail. And if that fails, revert their from the code.

**Comment by gufoe** [comment]
Hey contributors, I need your permission to relicence this project with a MIT licence.
Is it ok for you?

@awamper
@armonge
@prahal
@uhon
@juangl
@anassahmed
@stewiefox

Thanks everybody!

**Comment by prahal** [comment]
Yes, I agree to relicense my contributions under a MIT licence. They are personal contributions done on my own time.

**Comment by uhon** [comment]
Sure, go for it!

**Comment by armonge** [comment]
Go ahead
